src/libraries/System.Runtime/tests/System/GenericMathTests.cs

```diff
@@ -1,73 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System.Collections.Generic;
-using System.Linq;
-using Xunit;
-
-namespace System.Tests
-{
-    public static class GenericMath
-    {
-        public static TResult Average<TSelf, TResult>(IEnumerable<TSelf> values)
-            where TSelf : INumber<TSelf>
-            where TResult : INumber<TResult>
-        {
-            TResult sum = Sum<TSelf, TResult>(values);
-            return TResult.Create(sum) / TResult.Create(values.Count());
-        }
-
-        public static TResult StandardDeviation<TSelf, TResult>(IEnumerable<TSelf> values)
```

Comment:
Why delete these?

Reply:
Accident. Must have deleted these by mistake when adding in the other tests.